configs/permissions/notifications.json

```diff
@@ -1,15 +1,15 @@
 {
-    "*": [
-        {
-            "verb": "*"
-        }
-    ],
     "notifications": [
         {
             "verb": "read"
         },
         {
             "verb": "write"
         }
+    ],
+    "*": [
```

Comment:
Is it documented somewhere that in such a case the specific case has precedence over `"*"` ?